### DISCUSSION TITLE
I am looking for .NET sample

### DISCUSSION THREAD

**Discussion by kondareddy** [sipsorcery-org/sipsorcery#1158] [Q&A]
I am looking for a sample where p2p sharing between web and C# wpf app. Source sharing is from web (react.js) and receiving is on C# WPF

Even if I can find downloadable code for below will be really useful.

https://www.videosdk.live/developer-hub/media-server/sipsorcery-webrtc

With latest versions below packages.
![image](https://github.com/user-attachments/assets/381adbf3-4d62-4721-961b-fe2de2aa6234)

I am not able to find this_peerConnection.OnTrack

**Comment by sipsorcery** [comment]
The closest thing is the [data channel get started](https://github.com/sipsorcery-org/sipsorcery/blob/master/examples/WebRTCExamples/WebRTCGetStartedDataChannel/Program.cs) command line app example.
